File: Python/2037.py
# 2037. Minimum Number of Moves to Seat Everyone
# https://leetcode.com/problems/minimum-number-of-moves-to-seat-everyone/

class Solution:
  def minMovesToSeat(self, seats: List[int], students: List[int]) -> int:
    seats.sort()
    students.sort()

    out = 0

    for i in range(0, len(seats)):
      out += abs(seats[i] - students[i])

    return out

File: Rust/2037.rs
/**
 * 2037. Minimum Number of Moves to Seat Everyone
 * https://leetcode.com/problems/minimum-number-of-moves-to-seat-everyone/
 */

impl Solution {
  pub fn min_moves_to_seat(seats: Vec<i32>, students: Vec<i32>) -> i32 {
    let mut seats = seats;
    let mut students = students;

    seats.sort();
    students.sort();

    let mut out: i32 = 0;

    for i in 0..seats.len() {
      out += (seats[i] - students[i]).abs();
    }

    return out;
  }
}
